CanHazFunny/CanHazFunny.Tests/JesterTests.cs

```diff
@@ -1,10 +1,36 @@
 using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using System.Diagnostics.Metrics;
 
 namespace CanHazFunny.Tests
 {
     [TestClass]
     public class JesterTests
     {
-        
+        [TestMethod]
+        public void JokeService_TellsJoke()
```

Comment:
```suggestion
        public void JokeService_TellsJoke()
```
public void JokeService_TellsJoke_IsNotNull()